### ISSUE TITLE
Gradle Task :sqflite:testDebugUnitTest FAILED

### ISSUE THREAD

**Issue by wooldridgetm** [tekartik/sqflite#74]
I'm attempting to build my Android Project from the Terminal & I'm getting an error *Task :sqflite:testDebugUnitTest FAILED*.

` > Task :sqflite:testDebugUnitTest FAILED`

`com.tekartik.sqflite.SqlCommandTest > sqlArguments FAILED`
     `java.lang.NoClassDefFoundError at SqlCommandTest.java:43`
        `Caused by: java.lang.ClassNotFoundException at SqlCommandTest.java:43`

<img width="1189" alt="android sqlite build error" src="https://user-images.githubusercontent.com/13317009/42290781-8421ad26-7f95-11e8-97f1-cc8d214bc970.png">

REPRODUCE:
1. create a flutter project & add sqflite as a dependency & run 'flutter packages get'.
2. Open Terminal, navigate to the project directory.
3. type 'cd android'
4. ./gradlew clean cleanBuildCache
5. ./gradlew build -i

```
> Task :sqflite:testDebugUnitTest FAILED
Putting task artifact state for task ':sqflite:testDebugUnitTest' into context took 0.0 secs.
Up-to-date check for task ':sqflite:testDebugUnitTest' took 0.163 secs. It is not up-to-date because:
  Task has failed previously.
Starting process 'Gradle Test Executor 1'. Working directory: /Users/wooldridgetm/.pub-cache/hosted/pub.dartlang.org/sqflite-0.11.0/android Command: /Library/Java/JavaVirtualMachines/jdk-10.0.1.jdk/Contents/Home/bin/java -Djava.awt.headless=true -Dorg.gradle.native=false @/private/var/folders/nc/jj8r90ts62v5mtk69fgzmx540000gn/T/gradle-worker-classpath430561954086607090txt -Dfile.encoding=UTF-8 -Duser.country=US -Duser.language=en -Duser.variant -ea worker.org.gradle.process.internal.worker.GradleWorkerMain 'Gradle Test Executor 1'
Successfully started process 'Gradle Test Executor 1'

com.tekartik.sqflite.SqlCommandTest > sqlArguments FAILED
    java.lang.NoClassDefFoundError: io/flutter/plugin/common/MethodChannel$MethodCallHandler
        at java.base/java.lang.ClassLoader.defineClass1(Native Method)
        at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1009)
        at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:174)
        at java.base/jdk.internal.loader.BuiltinClassLoader.defineClass(BuiltinClassLoader.java:801)
        at java.base/jdk.internal.loader.BuiltinClassLoader.findClassOnClassPathOrNull(BuiltinClassLoader.java:699)
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(BuiltinClassLoader.java:622)
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:580)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:190)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:499)
        at com.tekartik.sqflite.SqlCommand.toValue(SqlCommand.java:28)
        at com.tekartik.sqflite.SqlCommand.getSqlArguments(SqlCommand.java:108)
        at com.tekartik.sqflite.SqlCommand.getSqlArguments(SqlCommand.java:171)
        at com.tekartik.sqflite.SqlCommandTest.sqlArguments(SqlCommandTest.java:43)

        Caused by:
        java.lang.ClassNotFoundException: io.flutter.plugin.common.MethodChannel$MethodCallHandler
            at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:582)
            at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:190)
            at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:499)
            ... 13 more

4 tests completed, 1 failed

```

**Comment by wooldridgetm** [comment]
the HTML report is attached in a zip file for your convenience.

<img width="605" alt="screen shot 2018-07-04 at 2 50 07 pm" src="https://user-images.githubusercontent.com/13317009/42291453-b42f0dca-7f99-11e8-8525-b1770b1f4c88.png">

[html-test-report.zip](https://github.com/tekartik/sqflite/files/2164142/html-test-report.zip)

**Comment by alextekartik** [comment]
Thanks for the detailed report. I have a (very small) pure java unit test which indeed had some unfortunate link to some other flutter java code. After moving constants around, the issue should be fixed in the published 0.11.0+1 version

**Comment by wooldridgetm** [comment]
np. Glad you figure it out. I'll check out the fix soon & let you know. Thanks for the prompt response!
